airbyte-db/db-lib/src/main/resources/configs_database/schema_dump.txt

```diff
@@ -261,6 +262,7 @@ alter table "public"."workspace_service_account"
   add constraint "workspace_service_account_workspace_id_fkey"
     foreign key ("workspace_id")
     references "public"."workspace" ("id");
+comment on column "public"."actor_definition"."max_seconds_between_messages" is 'Define the number of seconds allowed between 2 messages emitted by the connector before ';
```

Comment:
Did the comment message get cut off?  Seems to be missing something after "before".